controllers/rhmi/rhmi_controller_test.go

```diff
@@ -0,0 +1,147 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+
+	threescalev1 "github.com/3scale/3scale-operator/pkg/apis/apps/v1alpha1"
+	monitoringv1 "github.com/integr8ly/application-monitoring-operator/pkg/apis/applicationmonitoring/v1alpha1"
+	crov1 "github.com/integr8ly/cloud-resource-operator/apis/integreatly/v1alpha1"
+	kafkav1alpha1 "github.com/integr8ly/integreatly-operator/apis-products/kafka.strimzi.io/v1alpha1"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	keycloak "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	cloudcredentialv1 "github.com/openshift/api/operator/v1"
+	projectv1 "github.com/openshift/api/project/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	usersv1 "github.com/openshift/api/user/v1"
+	coreosv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1"
+	operatorsv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+const (
+	operatorNamespace = "openshift-operators"
+)
+
+func getBuildScheme() (*runtime.Scheme, error) {
+	scheme := runtime.NewScheme()
+	err := threescalev1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = keycloak.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = integreatlyv1alpha1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = operatorsv1alpha1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = corev1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = coreosv1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = kafkav1alpha1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = usersv1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = oauthv1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = routev1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = projectv1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = crov1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = monitoringv1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = apiextensionv1.SchemeBuilder.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	err = cloudcredentialv1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+
+	return scheme, err
+}
```

Comment:
You don't need to add all this packages to schema.
The test will only use `cloudcredentialv1`.

Reply:
Done. Thank you for comment.